app/javascript/chat/actions/actions.js

```diff
@@ -1,15 +1,16 @@
 import { createDataHash } from '../util';
+import { request } from '../../utilities/http';
 
-export function getAllMessages(channelId, messageOffset, successCb, failureCb) {
-  fetch(`/chat_channels/${channelId}?message_offset=${messageOffset}`, {
-    Accept: 'application/json',
-    'Content-Type': 'application/json',
-    credentials: 'same-origin',
-  })
-    .then((response) => response.json())
-    .then(successCb)
-    .catch(failureCb);
-}
+// export function getAllMessages(channelId, messageOffset, successCb, failureCb) {
+//   fetch(`/chat_channels/${channelId}?message_offset=${messageOffset}`, {
+//     Accept: 'application/json',
+//     'Content-Type': 'application/json',
+//     credentials: 'same-origin',
+//   })
+//     .then((response) => response.json())
+//     .then(successCb)
+//     .catch(failureCb);
+// }
 
 export function sendMessage(messageObject, successCb, failureCb) {
```

Comment:
refactor (non-blocking): Consider making this an async function as it makes it clearer to a consumer of the function that a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) is being returned.

```suggestion
export async function sendMessage(messageObject, successCb, failureCb) {
```